### PR TITLE
Add positions and exclude-positions flags for vcftools

### PR DESCRIPTION
**Pull request by gq1**
## PR checklist

Closes #XXX 

- [ ] This comment contains a description of changes (with reason).
- [ ] If you've fixed a bug or added code that should be tested, add tests!
- [ ] If you've added a new tool - have you followed the module conventions in the [contribution docs](https://github.com/nf-core/modules/tree/master/.github/CONTRIBUTING.md)
- [ ] If necessary, include test data in your PR.
- [ ] Remove all TODO statements.
- [ ] Emit the `versions.yml` file.
- [ ] Follow the naming conventions.
- [ ] Follow the parameters requirements.
- [ ] Follow the input/output options guidelines.
- [ ] Add a resource `label`
- [ ] Use BioConda and BioContainers if possible to fulfil software requirements.
- Ensure that the test works with either Docker / Singularity. Conda CI tests can be quite flaky:
  - [ ] `PROFILE=docker pytest --tag <MODULE> --symlink --keep-workflow-wd --git-aware`
  - [ ] `PROFILE=singularity pytest --tag <MODULE> --symlink --keep-workflow-wd --git-aware`
  - [ ] `PROFILE=conda pytest --tag <MODULE> --symlink --keep-workflow-wd --git-aware`

### CODE REVIEW

File: modules/nf-core/vcftools/main.nf [hunk 1]
```diff
@@ -91,10 +91,14 @@ process VCFTOOLS {
 
     def bed_arg  = (args.contains('--bed')) ? "--bed ${bed}" :
         (args.contains('--exclude-bed')) ? "--exclude-bed ${bed}" :
-        (args.contains('--hapcount')) ? "--hapcount ${bed}" : ''
+        (args.contains('--hapcount')) ? "--hapcount ${bed}" :
+        (args.contains('--positions')) ? "--positions ${bed}" :
+        (args.contains('--exclude-positions')) ? "--exclude-positions ${bed}"  : ''
```
Comment (maxulysse) [inline modules/nf-core/vcftools/main.nf:96]: Are the different args exclusive?

Comment (gq1) [reply modules/nf-core/vcftools/main.nf:96]: Yes, I think so.

**Comment by gq1** [comment]
Two tests tests failed because I just re-pulled the upstream.

File: modules/nf-core/vcftools/main.nf (hunk 1 above)
Comment (gq1) [reply modules/nf-core/vcftools/main.nf:96]: @maxulysse Hi, Maxime. By any chance you can approve this and merge it? Thanks.

**Review by SPPearce — APPROVED** [review]
I think the changes here are fine.

We are moving to nf-test over pytest, and  currently the linting check is set to mandate this if you pull upstream. That is under discussion currently, but any chance you can swap to nf-test?

**Comment by muffato** [comment]
@SPPearce : 👍🏼 we'll look at moving to nf-test in a different PR

**Pull request merged**